app/front/scripts/controllers/provide-metadata.js

```diff
@@ -25,36 +32,41 @@ angular.module('Application')
           end: fiscalPeriod ? fiscalPeriod.end : ''
         };
 
-        $scope.$watch('attributes.title', function() {
```

Comment:
Because it is redundant: we already have the same watcher (several lines below) that deeply watches entire `attributes` property.